docs/running-on-yarn.md

```diff
@@ -461,15 +461,28 @@ To use a custom metrics.properties for the application master and executors, upd
   </td>
 </tr>
 <tr>
-  <td><code>spark.yarn.security.tokens.${service}.enabled</code></td>
+  <td><code>spark.yarn.security.credentials.${service}.enabled</code></td>
   <td><code>true</code></td>
   <td>
-  Controls whether to retrieve delegation tokens for non-HDFS services when security is enabled.
-  By default, delegation tokens for all supported services are retrieved when those services are
+  Controls whether to obtain credentials for services when security is enabled.
+  By default, credentials for all supported services are retrieved when those services are
   configured, but it's possible to disable that behavior if it somehow conflicts with the
   application being run.
   <p/>
-  Currently supported services are: <code>hive</code>, <code>hbase</code>
+  Currently supported services are: <code>hdfs</code>, <code>hive</code>, <code>hbase</code>
+  <p/>
+  For other 3rd party services need to be communicate with Spark through security way should implement
```

Comment:
a few grammar things here and I think we should separate this out into the "Running in a Secure Cluster" section.

It also has a few places that need updated from tokens. to credentials.